src/integration/resources/common/coordinator_client.go

```diff
@@ -0,0 +1,559 @@
+// Copyright (c) 2021  Uber Technologies, Inc.
```

Comment:
This is mostly moved from `src/integration/resources/docker/coordinator.go` so that Docker and in-process coordinator impls can share how they make coordinator calls. GH doing a terrible job diffing here.